arches/install/arches-templates/project_name/templates/index.htm

```diff
@@ -69,7 +69,6 @@
                             <h1>{% blocktrans %}Arches | {{version}}{% endblocktrans %}</h1>
                         </div>
                     </div>
-                </div>
```

Comment:
By my count, this div removal causes an imbalance.  It's worth noting that this div is also present in the main htm file.